proto/rpc/user/user.proto

```diff
@@ -133,6 +136,14 @@ service UserService {
     };
   };
 
+  rpc UpdateUserForceReset(paralus.dev.types.user.v3.User)
```

Comment:
do we need user type as input / output ?

Reply:
Removed extra overhead and updated the endpoint